tests/ui/precondition-checks/read_volatile.rs

```diff
@@ -1,7 +1,7 @@
 //@ run-fail
 //@ compile-flags: -Copt-level=3 -Cdebug-assertions=no -Zub-checks=yes
 //@ error-pattern: unsafe precondition(s) violated: ptr::read_volatile requires
-//@ revisions: null misaligned
+//@ revisions: misaligned
 
 #![allow(invalid_null_arguments)]
```

Comment:
I just noticed the line above. I assume it's unused now, since there is no longer a null argument, and if so, I'll remove it. But more importantly, does this mean there is still a check enabled by default that prevents the null usage? I think it may be related to the source snippet below.

https://github.com/rust-lang/rust/blob/2398bd60ef526e686a38a299cc2fa991d8b3c33e/compiler/rustc_lint/src/ptr_nulls.rs#L160-L175

Should I remove `read_volatile`/`write_volatile`?